src/infrastructure/validation/logger/FileLogger.java

```diff
@@ -0,0 +1,108 @@
+package infrastructure.validation.logger;
+
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.PrintWriter;
+import java.io.IOException;
+
+/**
+ * FileLogger class that is part of loggerManager,
+ * and used for logging to the a file (the log file)
+ * Implements ILogger interface
+ * 
+ * @author Navaneeth M Nambiar
+ */
+public class FileLogger implements ILogger {
+
+	/** stores the format specified for the time-stamp */
+	private DateTimeFormatter timeStampFormat;
+		
+	/** string that holds the location of the log file */
+	private static String logFile;
+	
+	/**
+	 *  constructor for FileLogger
+	 *  protected type since it needs to be only invoked by LoggerManager class
+	 *  @see logger.LoggerManager
+	 */
```

Comment:
I don't think @see is valid block tag

Reply:
I mean javadoc and eclipse do seem to support it. It's fine if you want it to be removed too

Reply:
Okay, it can be used